### DISCUSSION TITLE
Screen capturing all displays

### DISCUSSION THREAD

**Discussion by imeszaros** [awawa-dev/HyperHDR#885] [Q&A]
Hi all,

i have two displays, with a single LED strip running along each side of each display, controlled by WLED on an ESP32.

Currently I'm using it with Prismatik, but would like to try HyperHDR. Is it possible to capture both displays at the same time?

Thanks in advance.

**Comment by awawa-dev** [comment]
Hi
That currently depends on the OS - why don't you try yourself?

**Reply by imeszaros** [reply]
Hi! I'm on Windows 11. At the screen capture settings I can only choose one of my displays but not both of them.

**Reply by awawa-dev** [reply]
Then you will need https://github.com/sabaatworld/HyperionScreenCap or USB grabber for the second monitor on Windows

**Comment by awawa-dev** [comment]
Hi @imeszaros 
Added support for multiple monitors in Windows. Feel free to test.

PR:
https://github.com/awawa-dev/HyperHDR/pull/966
Installers:
https://github.com/awawa-dev/HyperHDR/actions/runs/11594672924
